style.css

```diff
@@ -34,8 +34,8 @@ body {
    font-family: 'Over the Rainbow' !important;
    color: #1E90FF !important;
    position: relative;
-   height: 0 auto !important;
-   width: 0 auto !important;
+   height: 0 auto;
```

Comment:
Height of 0 auto does not do anything. I'm pretty sure if you delete this line it will be exactly the same